recipes-core/systemd/systemd_%.bbappend

```diff
@@ -20,5 +19,7 @@ do_install_append() {
     done
 }
 
+EXTRA_OECONF = "myhostname"
```

Comment:
It looks to me like this means "-Dmyhostname=true" is passed to Meson when systemd is built, which means "nss-myhostname support".

What you really want, I think is for "-Dhostnamed=true" ("install the systemd-hostnamed stack") to be passed, which means this should read:

EXTRA_OECONF = "hostnamed"

Should it not also be an append (+=)?

And hum... should it not be appended to PACKAGECONFIG? (Is EXTRA_OECONF even relevant here? Only for configure and should be explicit configure flag? Since systemd is built with Meson it should be EXTRA_OEMESON if anything?)